tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,81 @@
+"""Task 1 and 2 of Module 2."""
+
+from collections import defaultdict
+# create CONSTANT variables for weight
```

Comment:
Those notes look more like self TODO notes, not sure if we need them in the final version? :)